llama_hub/microsoft_sharepoint/README.md

```diff
@@ -20,12 +20,18 @@ More info on Microsoft Graph APIs - [Refer here](https://learn.microsoft.com/en-
 
 To use this loader `client_id`, `client_secret` and `tenant_id` of the registered app in Microsoft Azure Portal is required.
 
+This loader can:
+- Load files present in a specific folder in SharePoint
+- Load all files present in the drive of a SharePoint
+- Load all pages under a SharePoint site
+
```

Comment:
Line no 28 can be removed as you have already mentioned this as the first point!